remo/profiles/templates/profiles_people.jinja

```diff
@@ -97,6 +97,13 @@
       </form>
     </div>
   </div>
+  
+  {% if user.is_authenticated() %}
```

Comment:
I know this was in there already before. I think we can remove this check, I don't see why we should only show this for authenticated users.

Reply:
I think that is more for privacy to not know how many reps are involved. 
I will remove that now.